Detections/CommonSecurityLog/Zscaler-LowVolumeDomainRequests.yaml

```diff
@@ -19,25 +19,31 @@ relevantTechniques:
 query: |
 
   let timeRange = 1d;
+  let scriptExtensions = dynamic([".php", ".aspx", ".asp", ".cfml"]);
   //The number of URI's seen to be suspicious, higher = less likely to be suspicious
   let uriThreshold = 1;
-  let suspiciousURLs =
   CommonSecurityLog
   | where TimeGenerated >= ago(timeRange)
-  // Only look at connections that were allowed through the web proxy. Comment line out for all events including those which were blocked
+  // Only look at connections that were allowed through the web proxy
   | where DeviceVendor =~ "Zscaler" and DeviceAction =~ "Allowed"
-  // Only look where some data was exchanged. Comment out to see 0 byte connections
+  // Only look where some data was exchanged.
   | where SentBytes > 0 and ReceivedBytes > 0
   // Extract the Domain
   | extend Domain = iff(countof(DestinationHostName,'.') >= 2, strcat(split(DestinationHostName,'.')[-2], '.',split(DestinationHostName,'.')[-1]), DestinationHostName)
-  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), makelist(RequestURL), makelist(DestinationIP), makelist(SourceIP), numOfConnections = count() by Domain
+  | extend GetData=iff(RequestURL in "?", 1, 0)
```

Comment:
If you are using "in", then this must be in parenthesis - iff(RequestURL in **("?")**, 1, 0).  However, you are only matching on "?" and so you should use "=" instead of "in".